# K: 21
# # K_normal_estimation: 10
# epsilon: 0.25
# K=29, eps=0.3 works pretty good for angles
K: 11
epsilon: 0.3
# half disc - quite good for 35:0.3 && 11:0.2

K_2: 11
epsilon_2: 0.2

# abs_path: "/home/honzuna/"
# file_name: "pointcloud.pcd"
# file_name: "sphere.pcd"
# file_name: "plane.pcd"
# terrain novy malin
# file_name: "workspace/src/aerial_core_planning/ros_packages/mrs_octomap_planner/maps/pcds"
# file_name: ".ros/scan.pcd"

abs_path: "/home/honzuna/moje/bakalarka/modely/naki_gazebo_resources-master/models/"
# file_name: "cholina/pcd/cholina.pcd"  # 873 273
# file_name: "kn_backyard/pcd/kn_backyard.pcd"  # 4 511 074
# file_name: "kn_gym_A1/pcd/kn_gym_A1.pcd"  # 233 223
# file_name: "koprivna/pcd/koprivna.pcd"  # 411 077

file_name: "novy_malin/pcd/novy_malin.pcd"  # 653 652 # FOR SIMULATION
# file_name: "olomouc_marie_snezna/pcd/olomouc_marie_snezna.pcd"  # 3 603 056
# file_name: "olomouc_sv_moric/pcd/olomouc_sv_moric.pcd"  # 1 000 000
# file_name: "prostejov/pcd/prostejov.pcd"  # 2 105 238
# file_name: "smockovi/pcd/smockovi_court.pcd"  # 82 548
# file_name: "stara_voda/pcd/stara_voda.pcd"  # 778 085
# file_name: "sumperk/pcd/sumperk.pcd"  # 548 000
# file_name: "uherske_hradiste/pcd/uherske_hradiste.pcd"  # 2 580 696
# file_name: "zabreh/pcd/zabreh.pcd"  # 1 954 927

# file_name: "planning_tower/pcd/power_tower_danube.pcd"

trashold: 0.2
k_angle: 0.5
k_halfdisc: 0.5
k_shape: 0.0

weight_boundary: 20
minimum_loop_length: 18

q: 0

# RANSAC parameters

n_ransac: 3
k_ransac: 2000
#t_ransac: 0.05
t_ransac: 0.25
d_ransac: 1

uav_min_distance: 1
uav_max_distance: 2
uav_min_angle: 60
uav_min_area_width: 0.6
uav_sample_width: 1.5
uav_sample_height: 1

area_samples_first: 1
area_samples_second: 1
area_samples_mid: 1

# UAV POSITIONS
uav_start_x: 0.0
uav_start_y: 0.0
uav_start_z: 2.0
uav_start_heading: 0.0

uav_end_x: 0.0
uav_end_y: 0.0
uav_end_z: 2.0
uav_end_heading: 0.0

listen: false

# ===== for kn_gym_A1 to detect two holes =====
# K: 11
# epsilon: 0.3
# K_2: 11
# epsilon_2: 0.2
# trashold: 0.2
# k_angle: 0.5
# k_halfdisc: 0.5
# k_shape: 0.0
# weight_boundary: 1.8
# minimum_loop_length: 5 or 18 for d = weight instead of length
# n_ransac: 3
# k_ransac: 1000
# t_ransac: 0.05
# d_ransac: 1
# uav_min_distance: 1
# uav_max_distance: 2
# uav_min_angle: 60
# uav_min_area_width: 0.6
# uav_sample_width: 1.5
# uav_sample_height: 1

# NOVY_MALIN 5/8 děr chceme, jedna chybí
# K: 31
# epsilon: 0.1
# K_2: 21
# epsilon_2: 0.1
# trashold: 0.2
# k_angle: 0.4
# k_halfdisc: 0.6
# k_shape: 0.0
# weight_boundary: 20
# minimum_loop_length: 20
# ---------------------- 5/7 ok, 1 nenalezena
# K: 31
# epsilon: 0.1
# K_2: 8
# epsilon_2: 0.1
# trashold: 0.2
# k_angle: 0.6
# k_halfdisc: 0.4
# k_shape: 0.0